hbase-server/src/main/java/org/apache/hadoop/hbase/master/procedure/RSProcedureDispatcher.java

```diff
@@ -249,6 +249,20 @@ protected class ExecuteProceduresRemoteCall implements RemoteProcedureResolver,
       "hbase.regionserver.rpc.retry.interval";
     private static final int DEFAULT_RS_RPC_RETRY_INTERVAL = 100;
 
+    /**
+     * Config to determine the retry limit while executing remote regionserver procedure. This retry
+     * limit applies to only specific errors. These errors could potentially get the remote
+     * procedure stuck for several minutes unless the retry limit is applied.
+     */
+    private static final String RS_REMOTE_PROC_RETRY_LIMIT =
+      "hbase.master.rs.remote.proc.retry.limit";
+    /**
+     * The default retry limit. Value = {@value}
+     */
+    private static final int DEFAULT_RS_REMOTE_PROC_RETRY_LIMIT = 5;
```

Comment:
Why 5? Why not 3? 
Let's have a comment on justification for the default value.

Reply:
Done.